src/client/Microsoft.Identity.Client/AppConfig/ApplicationConfiguration.cs

```diff
@@ -60,6 +61,8 @@ public string ClientVersion
 
         public WindowsBrokerOptions WindowsBrokerOptions { get; set; }
 
+        public BrokerOptions BrokerOptions { get; set; }

```

Comment:
Can we inject the options straight into the RuntimeBroker class? And maybe get rid of `WindowsBrokerOptions` as well (since we should always transform them to `RuntimeBroker`)

Reply:
I would soft deprecate WindowsBrokerOptions as it may be needed for backward compatibility i.e. not to break other apps completely.

Reply:
+1 on just deprecate. Plus it's still needed for UWP?